From ebd51f0750b13046ebbfcce0212be00443ad36db Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=D0=BD=D0=B0=D0=B1?= <nabijaczleweli@gmail.com>
Date: Mon, 18 May 2020 18:41:33 +0200
Subject: dupa6: Revert "dupa5: trans test v[12]"

This reverts commit 675568be1f6137c607cec369e31dd03bc42073f0.
This reverts commit 3e32e8237436357737307a48e3fdbddfe4958bcb.
---
 include/libuutil_impl.h | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/include/libuutil_impl.h b/include/libuutil_impl.h
index c342080f..9e89889a 100644
--- a/include/libuutil_impl.h
+++ b/include/libuutil_impl.h
@@ -33,13 +33,9 @@
 #include <pthread.h>
 
 #include <sys/avl_impl.h>
-//#include  <sys/isa_defs.h>
+#include <sys/isa_defs.h>
 #include <sys/byteorder.h>
 
-#ifdef _SYS_ISA_DEFS_H
-#error ipipoo
-#endif
-
 #ifdef	__cplusplus
 extern "C" {
 #endif
-- 
2.26.2

